lib/src/functions/math.dart

```diff
@@ -140,6 +140,7 @@ final _hypot = BuiltInCallable("hypot", r"$numbers...", (arguments) {
 
 final _log = BuiltInCallable("log", r"$number, $base: null", (arguments) {
   var number = arguments[0].assertNumber("number");
+
```

Comment:
Style nit: I don't think these newlines necessarily improve readability; having every statement separated by an empty line doesn't really provide any more information than having no empty lines would.

Generally speaking, I think clustering a variable assignment in the same vertical block as a validity check for that variable, or that uses that variable, is good.